pallets/parachain-staking/src/lib.rs

```diff
@@ -147,7 +147,13 @@ pub mod pallet {
 	pub struct Collator<AccountId, Balance> {
 		pub id: AccountId,
 		pub bond: Balance,
-		pub nominators: OrderedSet<Bond<AccountId, Balance>>,
+		/// Set of all nominator AccountIds (to prevent >1 nomination per AccountId)
+		pub nominators: OrderedSet<AccountId>,
+		/// Top T::MaxNominatorsPerCollator::get() nominators
+		pub top_nominators: OrderedSet<Bond<AccountId, Balance>>,
+		/// Bottom nominators (unbounded)
+		pub bottom_nominators: OrderedSet<Bond<AccountId, Balance>>,
+		/// Sum of top nominations + self.bond
 		pub total: Balance,
```

Comment:
TODO: 1 counter for counted_total and 1 counter for total locked in support of this nominator (which would include the sum of the bottom nominations as well)

Reply:
done